core/src/main/java/discord4j/core/spec/GuildCreateSpec.java

```diff
@@ -29,6 +29,19 @@
 import java.util.List;
 import java.util.function.Consumer;
 
+/**
+ * A spec used to configure and create a {@link Guild}.
+ * <b>This can only be used for bots in less than 10 guilds.</b>
```

Comment:
The newlines in this Javadoc are weird. Keep it consistent and just keep going until you hit the endline limit.